src/mind_agents/play.py

```diff
@@ -36,17 +37,43 @@
 
 
 def load_config() -> dict[str, str]:
-    """Load API keys from config file.
+    """Load API keys from environment variables or config file.
 
     Returns:
         Dict containing API keys for different providers
 
     Raises:
-        ValueError: If config file not found or has invalid format
+        ValueError: If neither environment variables nor config file are properly set up
     """
+    # First try environment variables
+    config = {}
+    env_keys = {
+        "openai_api_key": "OPENAI_API_KEY",
+        "anthropic_api_key": "ANTHROPIC_API_KEY",
+        "google_api_key": "GOOGLE_API_KEY",
+        "groq_api_key": "GROQ_API_KEY",
+    }
+    
+    # Check if all keys are available in environment
+    all_env_keys_present = True
```

Comment:
Wont this prevent users from running this if they only have some of the api keys setup?

Reply:
wow that's not good, I need to stop programming past 11, fixed, check below for partial configs to be allows:
373f66004ad8de171432f03dd4c83b45deaad0f2